src/ripple/app/tx/Transaction.cpp

```diff
@@ -40,13 +40,18 @@ Transaction::Transaction (STTx::ref sit, Validate validate)
         mTransactionID  = mTransaction->getTransactionID ();
         mAccountFrom    = mTransaction->getSourceAccount ();
     }
+    catch (std::exception& e)
+    {
+        reason = e.what();
+        return;
+    }
     catch (...)
     {
         return;
```

Comment:
Here you might set `reason` to something like `"Unexpected exception"`.  I seem to recall that Howard is not fond of `catch (...)`, but I understand that was pre-existing code.  I think it's prudent not to mess with that in this pull request.